src/sass/components/_tour.scss

```diff
@@ -29,3 +29,20 @@
   }
 }
 /* stylelint-enable declaration-no-important */
+
+// Override background image
+.slds-popover_walkthrough .slds-popover__header {
+  background-image: url('~img/popover-header.png');
+}
+
+.popover-trigger {
+  color: $palette-violet-65;
+}
+
+// Keep text from being truncated in table and page header
+.slds-popover {
+  font-size: $font-size-3;
```

Comment:
@dvdherron I don't see where this `font-size` setting is actually needed; can you point me to what this is overriding?

Reply:
@jgerigmeyer If you look at the popover text for "project name and github link" the text is smaller than the body size. and in the popover where the metecho logo is, the text is set larger, so there needed to be a way to get them all to have the same font-size.